apps/sms/views/conversation/js/conversation.js

```diff
@@ -516,6 +547,11 @@ var ConversationView = {
 
     var prevPanel = args.meta.prev;
 
+    var emailThread = Settings.supportEmailRecipient &&
+      Threads.active.participants.some(Utils.isEmailAddress);
+
+    Compose.setupLock({ getType: () => emailThread ? 'mms' : null });
```

Comment:
it's fine to me; also I actually think we should return 'mms' for group mms threads...